utility.lua

```diff
@@ -57,6 +57,15 @@ local delay = function(x)
 	return (function() return x end)
 end
 
+-- Some containers consist of multiple nodes and only one have inventory.
+-- This is because minetest does not allow multiple nodes to have a common inventory.
+local real_inv_pos = function(target_pos)
+	if minetest.get_meta(target_pos):get_string("real_inv_pos") ~= "" then
```

Comment:
```Lua
local pos_str = minetest.get_meta(target_pos):get_string("real_inv_pos")
return pos_str ~= "" and minetest.string_to_pos(pos_str) or target_pos
```
Omits two calls.

Also: Which mod does set this meta field? A short comment would be very helpful to have to that it's not removed for optimization reasons..